### PR TITLE
Add button interaction, and modal for mobile

### PR DESCRIPTION
**Pull request by Codooze**
# In this milestone
- worked on button interaction using js
- implemented a nav for the mobile version

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi **Team**, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters errors, kindly make sure all the linters are passing on your machine and kindly add the ESlint ([JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript)) to your project to check your Javascript code.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -21,6 +27,23 @@
...
+            <a class="mobile-menu-option" href="#about_main_container">About</a>
+          </li>
+          <li>
+            <a class="mobile-menu-option" href="#contact_form_container"
+              >Contact</a
+            >
+          </li>
+        </ul>
+      </div>
+
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly make sure that when your application loads the mobile menu is not visible on the desktop version, please see the recording below to see how your app behaves 👍🏿 


https://github.com/Mehr-o-maah/Portfolio_2.github.io/assets/69638013/8aa851ad-a0f1-4857-9418-dc75004d8ceb

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Mehr-o-maah and @JavierAybar 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- All functionalities for on click events are implemented ✅  



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const menuHamburger = document.querySelector('.menu');
+
+mobileMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('menu-mobile-active');
+  menuHamburger.classList.remove('menu-hidden');
+});
+
+menuHamburger.addEventListener('click', () => {
+  menuHamburger.classList.toggle('menu-hidden');
+  mobileMenu.classList.toggle('menu-mobile-active');
```
Comment (shella12) [inline main.js:11]: - [ ] [OPTIONAL] Great job implementing the change requested by previous reviewer however please make sure that when the mobile menu is active/displayed and we expand back to the desktop screen size the menu disappears. In other words mobile menu should never appear on desktop screen size for better user experience and consistency

Screenshot 2023-06-20 at 7 41 10 AM

**Review by shella12 — APPROVED** [review]
Hi @Mehr-o-maah, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great looking portfolio ✔️ 
- Clean design implementation 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**